src/game/construct_types.h

```diff
@@ -16,5 +16,10 @@ struct ConstructData {
 namespace Constructs {
     const static ConstructData CHEST = { ConstructType::CHEST, 50, "Chest", "A treasure chest full of gold.", "test.png" };
     const static ConstructData SPIKES = { ConstructType::SPIKES, 50, "Spikes", "It was me, spikes.", "dio.jpg" };
-    const static ConstructData REMOVE = { ConstructType::REMOVE, 0, "Remove", "Remove a block.", "Tomato.jpg" };
+    const static ConstructData REMOVE = { ConstructType::REMOVE, 200, "Remove", "Remove a block.", "Tomato.jpg" };
+    const static std::map<ConstructType, ConstructData> CONSTRUCTS = {
```

Comment:
This seems kind of awful. Can we make each `Construct` carry its relevant `ConstructData` instead?